rental.rb

```diff
@@ -0,0 +1,17 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] You are supposed to define Getters and Setters for @book and @person instance variables.
`attr_accessor :date, :book, :person`
The getter method allows you to access the value of the @book and @person instance variables from outside the class, while the setter method allows you to assign a new value to the @book and @person instance variables.

Reply:
thanks, that's done!